### ISSUE TITLE
iOS should use asset catalogs

### ISSUE THREAD

**Issue by ofZach** [openframeworks/openFrameworks#5894]
we are using a few icons (1x 2x) and launch screens but it's now preferred to use an asset catalog and on certain devices things like the keyboard are stretched since the sizing of the app is wrong: 

https://stackoverflow.com/questions/42937996/xcode-app-keyboard-looks-stretched?rq=1 

There's a tool called asset catalog creator pro that's pretty good and we can use to make a more modern set of images for launch and remove the older ones we have from the data folder, etc.

**Comment by danoli3** [comment]
Yeah it should already be using the asset catalogs. Did you use the project generator to make a project?

I may need an update though for new devices.
It can be found in scripts/templates/ios/mediaAssets

**Comment by dimitre** [comment]
Launchscreen was recently modernized, now it uses an svg file instead.
I've submitted a PR to remove the bin/data images 
https://github.com/openframeworks/openFrameworks/pull/7033
